code/obj/table.dm

```diff
@@ -770,6 +770,96 @@ TYPEINFO_NEW(/obj/table/reinforced/chemistry)
 	has_drawer = TRUE
 	drawer_contents = list(/obj/item/reagent_containers/glass/beaker = 7)
 
+/obj/table/reinforced/chemistry/basicsup
+	name = "basic supply lab counter"
+	desc = "Everything an aspiring chemist needs to start making chemicals!"
+	has_drawer = TRUE
+	drawer_contents = list(/obj/item/paper/book/from_file/pharmacopia,
+				/obj/item/storage/box/beakerbox = 2,
+				/obj/item/reagent_containers/glass/beaker/large = 2,
+				/obj/item/clothing/glasses/spectro,
+				/obj/item/device/reagentscanner = 2,
+				/obj/item/reagent_containers/dropper/mechanical = 2,
+				/obj/item/reagent_containers/dropper = 2)
+
+/obj/table/reinforced/chemistry/auxsup
+	name = "auxiliary supply lab counter"
+	desc = "Extra supplies for the discerning chemist."
+	has_drawer = TRUE
+	drawer_contents = list(/obj/item/storage/box/beakerbox,
+				/obj/item/storage/box/patchbox,
+				/obj/item/storage/box/syringes,
+				/obj/item/reagent_containers/food/drinks/fueltank,
+				/obj/item/reagent_containers/glass/beaker/large,
+				/obj/item/clothing/glasses/spectro = 2,
+				/obj/item/device/reagentscanner = 2,
+				/obj/item/reagent_containers/dropper/mechanical,
+				/obj/item/reagent_containers/iv_drip = 2)
```

Comment:
```suggestion
				/obj/item/reagent_containers/glass/beaker/large,
				/obj/item/clothing/glasses/spectro = 2,
				/obj/item/device/reagentscanner = 2,
				/obj/item/reagent_containers/dropper/mechanica)
```

Reply:
@Tarmunora, Are the IV drip/fuel tank here something that could be reinstated as a separate proposal if received well?